recipe/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: 8535529ddb807067b0d59661dce1d9a6ddb2a218398a38ea7772328ad8a6ea13
 
 build:
-  number: 1
-  skip: true  # solvers are missing on:  # [win and py==27]
+  number: 2
+  skip: true  # [py==27 and (win or (not x86_64))]
```

Comment:
```suggestion
  skip: true  # [py==27 and win]
  skip: true  # [py==27 and not x86_64]
```

Reply:
Just makes the logic cleaner.
But soon it won't be necessary (I hope)
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/306